helmfile/charts/s3-exporter/values.yaml

```diff
@@ -0,0 +1,72 @@
+# Default values for s3-exporter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+s3:
+  region: ""
+  regionEndpoint: ""
+  accessKey: ""
+  secretKey: ""
+  buckets: []
+    # - my-bucket
+    # - second-bucket
+
+serviceMonitor:
```

Comment:
Maybe add a `seviceMonitor.enabled` here? Although for our use case I guess we never want do disable the service monitor.

On that note, if this we should host this chart. Doesn't it fit better if we create and upload it to https://github.com/elastisys/chart-repo?

Reply:
Good point! I didn't think about it actually, but I could definitely add it there. This chart is pretty generic so I guess it could be useful to separate it from ck8s On the other hand, we have https://github.com/elastisys/compliantkubernetes-apps/issues/355 that seems to be about just vendoring the charts so I'm not sure. But I do like the idea of having the charts in the chart-repo so I'll add a PR there and remove it from here unless I hear objections :slightly_smiling_face:

Reply:
I think we can keep that chart in this repo for the tmie being :)

Reply:
I looked into this a bit and realized that the chart-repo is for publishing the charts, not for "development", right? So I'll keep it here for now but we can still publish it if we want to. Later we could create a separate repo for the chart also if we want to decouple it.